<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="7" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-13T15:35:38 IST" name="Suite" finished-at="2022-08-13T15:35:52 IST" duration-ms="13926">
    <groups>
    </groups>
    <test started-at="2022-08-13T15:35:38 IST" name="Test" finished-at="2022-08-13T15:35:52 IST" duration-ms="13926">
      <class name="tests.LoginTest">
        <test-method is-config="true" signature="init()[pri:0, instance:tests.LoginTest@34123d65]" started-at="2022-08-13T15:35:26 IST" name="init" finished-at="2022-08-13T15:35:38 IST" duration-ms="12185" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method is-config="true" signature="initObject()[pri:0, instance:tests.LoginTest@34123d65]" started-at="2022-08-13T15:35:38 IST" name="initObject" finished-at="2022-08-13T15:35:38 IST" duration-ms="30" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initObject -->
        <test-method signature="verifybuttontext()[pri:1, instance:tests.LoginTest@34123d65]" started-at="2022-08-13T15:35:38 IST" name="verifybuttontext" finished-at="2022-08-13T15:35:38 IST" duration-ms="51" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [LOGIN] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [LOGIN] but found []
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at tests.LoginTest.verifybuttontext(LoginTest.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifybuttontext -->
        <test-method signature="login()[pri:2, instance:tests.LoginTest@34123d65]" started-at="2022-08-13T15:35:38 IST" name="login" finished-at="2022-08-13T15:35:47 IST" duration-ms="8494" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginTest@34123d65]" started-at="2022-08-13T15:35:52 IST" name="tearDown" finished-at="2022-08-13T15:35:52 IST" duration-ms="180" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.HomePageTest">
        <test-method is-config="true" signature="initObject()[pri:0, instance:tests.HomePageTest@33d512c1]" started-at="2022-08-13T15:35:47 IST" name="initObject" finished-at="2022-08-13T15:35:47 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initObject -->
        <test-method signature="goToTab()[pri:1, instance:tests.HomePageTest@33d512c1]" started-at="2022-08-13T15:35:47 IST" name="goToTab" finished-at="2022-08-13T15:35:49 IST" duration-ms="2918" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToTab -->
        <test-method signature="logout()[pri:2, instance:tests.HomePageTest@33d512c1]" started-at="2022-08-13T15:35:49 IST" name="logout" finished-at="2022-08-13T15:35:52 IST" duration-ms="2168" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method signature="verifyTitle()[pri:3, instance:tests.HomePageTest@33d512c1]" started-at="2022-08-13T15:35:52 IST" name="verifyTitle" finished-at="2022-08-13T15:35:52 IST" duration-ms="19" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitle -->
      </class> <!-- tests.HomePageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
